### PR TITLE
Created Contact Us Page

### LINKED ISSUE #186: Create Contact Us page
## Description
As a developer,
I need create a contact us page.

## Mockups
contact

### Update [Required]
- [x] contact
- [x] contact.component.html
- [x] contact.component.scss
- [x] contact.component.ts

## Definition of Done
- [x] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
6 hours 

@ancyp @divyanshu-rawat @Janiceilene Can you please assign me the issue :)

### PR DESCRIPTION
**Pull request by BaaniLeen**
### Description
Created Contact us Page for systers.io.
Can be merged after PR #185 

Fixes #186 

### Type of Change:

- Code
- Quality Assurance
- User Interface
- Outreach

**Code/Quality Assurance Only**
- New feature (non-breaking change which adds functionality pre-approved by mentors)

#### Mockups

screen shot 2018-05-28 at 11 46 05 am

screen shot 2018-05-28 at 11 46 13 am

This has been tested locally on my computer.

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
  
**Code/Quality Assurance Only**
- [x] My changes generate no new warnings 
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been published in downstream modules

### CODE REVIEW

File: src/app/contact/contact.component.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+			<div class = "col-md-4">
+				<a href="https://github.com/systers/communities/"><button type="button"  class="btn btn-rounded waves-light brand-c btn-lg btn-block" mdbWavesEffect style="background-color: #54BCEB;width:50%;margin: auto;" >Github</button></a>
+			</div>
+			<div class = "col-md-4">
+				<a href="http://systers.org/mailman/listinfo/systers-opensource"><button type="button" class="btn btn-rounded waves-light brand-c btn-lg btn-block" mdbWavesEffect style="background-color: #54BCEB;width:50%; margin: auto;">Mailing List</button></a>
+			</div>
+		</div>
+	</div>
+
+	<br><br><br>
```
Comment (ancyp) [inline src/app/contact/contact.component.html]: br tags is not good practice

File: e2e/protractor.conf.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  capabilities: {
+    "browserName": "chrome"
+  },
+  directConnect: true,
+  baseUrl: "http://localhost:4200/",
+  framework: "jasmine",
+  jasmineNodeOpts: {
+    showColors: true,
+    defaultTimeoutInterval: 30000,
+    print: function() {}
```
[the remark this answers is not in the document]
Comment (BaaniLeen) [reply e2e/protractor.conf.js]: Resolved :)

File: src/app/contact/contact.component.html (hunk 1 above)
Comment (BaaniLeen) [reply src/app/contact/contact.component.html]: Resolved by improving on the code modularity. Also aligned all the contact numbers :)

**Comment by BaaniLeen** [comment]
@ancyp The changes have been made. Can you please review the PR once again? :)

**Comment by ancyp** [comment]
@BaaniLeen conflicts?

**Comment by BaaniLeen** [comment]
@ancyp On it. Give me 5 min :)

**Comment by BaaniLeen** [comment]
@ancyp @Janiceilene Merge conflicts resolved :)

**Comment by ancyp** [comment]
Taking a second look,
the phone icons look too big and unaligned with the text.
We repeat US two times - just having it in a single line makes more sense.
It would be better to have slack,github image button.
The concern about not having icon for mailing list - we could have one like this https://goecoserve.com/wp-content/uploads/2015/03/subscribe.jpg

@Janiceilene what do you think?

**Comment by Janiceilene** [comment]
Ooh, I like the idea of an envelope with the word subscribe for the mailing list! I agree about the phone icons and the slack/github icon buttons, especially with a mailing list icon like the one @ancyp linked.

**Comment by Janiceilene** [comment]
I think the three icons should stay at the top, along with something like "Collaborate with us" or some wording that indicates that if they click on one of these three links, they'll be actively participating with the organization.

**Pull request merged**